decentraland/unity-renderer
# feat: click own avatar opens passport

## What does this PR change?

`Fixes #3954 `

The passport should be opened when clicking over your avatar.
Only available in 3rd person.
The outline effect has been added.
Works behind a feature flag: `click_own_avatar_passport`

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/click-own-passport
2. Switch to 3rd person camera
3. Hover the mouse over your avatar
4. The outline effect should be visible
5. Click over your avatar
6. The passport should be opened with your own profile
7. Check that the rest of the world features keeps working as expected in 1st and 3rd person camera, specially clicking & hovering other objects with the mouse in scenes
8. Go to: https://play.decentraland.org/?explorer-branch=feat/click-own-passport
9. The feature should be disabled

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
